src/Aks/readme.md

```diff
@@ -0,0 +1,206 @@
+- [NodeJS LTS](https://nodejs.org) (10.15.x LTS preferred)
+  - **Note**: It *will not work* with Node < 10.x. Using 11.x builds may cause issues as they may introduce instability or breaking changes.
+> If you want an easy way to install and update Node, [NVS - Node Version Switcher](../nodejs/installing-via-nvs.md) or [NVM - Node Version Manager](../nodejs/installing-via-nvm.md) is recommended.
+- [AutoRest](https://aka.ms/autorest) v3 beta <br>`npm install -g autorest@beta`<br>&nbsp;
+- PowerShell 6.0 or greater
+  - If you don't have it installed, you can use the cross-platform npm package <br>`npm install -g pwsh`<br>&nbsp;
+- .NET Core SDK 2.0 or greater
+  - If you don't have it installed, you can use the cross-platform npm package <br>`npm install -g dotnet-sdk-2.2`<br>&nbsp;
+
+## Run Generation
+In this directory, run AutoRest:
+> `autorest`
+
+---
+### AutoRest Configuration
+> see https://aka.ms/autorest
+
+> Directives
+``` yaml
+metadata:
+  authors: Microsoft Corporation
+  owners: Microsoft Corporation
+  description: 'Microsoft Azure PowerShell: $(service-name) cmdlets'
+  copyright: Microsoft Corporation. All rights reserved.
+  tags: Azure ResourceManager ARM PSModule $(service-name)
+  companyName: Microsoft Corporation
+  requireLicenseAcceptance: true
+  licenseUri: https://aka.ms/azps-license
+  projectUri: https://github.com/Azure/azure-powershell
+
+directive:
+  - where:
+      subject: Operation
+    hide: true
+  - where:
+      parameter-name: SubscriptionId
+    set:
+      default:
+        script: '(Get-AzContext).Subscription.Id'
+```
+
+``` yaml
+directive:
+  - from: managedClusters.json
+    where: $.definitions.SubResource.properties
+    transform: >
+      return {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource ID."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The name of the resource that is unique within a resource group. This name can be used to access the resource."
+        },
+        "restype": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource type"
+        }
+      }
+```
+
+### General settings
+> Values
+
+``` yaml
+service-name: Aks
+powershell: true
+azure: true
+branch: master
+repo: https://github.com/Azure/azure-rest-api-specs/blob/$(branch)
+prefix: Az
+subject-prefix: ''
+module-name: $(prefix).$(service-name)
+namespace: Microsoft.Azure.PowerShell.Cmdlets.$(service-name)
+clear-output-folder: true
+output-folder: .
+aks: $(repo)/specification/containerservice/resource-manager/Microsoft.ContainerService
+input-file:
+- $(aks)/stable/2019-10-01/managedClusters.json
+
+module-version: 0.0.1
+title: AksClient
+
+directive:
+  - where:
+      subject: Operation
+    hide: true
+  - where: $.definitions.Identifier.properties
+    suppress: R3019
+  - where:
+      verb: New|Set|Remove|Get
```

Comment:
Wy remove the Get and Remove cmdlets?

Reply:
need to change base class of get and remove, so hide here.